### ISSUE TITLE
p-tag: tags are not accessible by tabbing

### ISSUE THREAD

**Issue by akashuwtw** [primefaces/primeng#14925]
### Describe the bug

`p-tag`s are not accessible by tabbing and does not receive keyboard focus

### Environment

OS: Windows 10

### Angular version

16.2.7

### PrimeNG version

16.5.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Browser(s)

Chrome 121.0.6167.161, Edge 121.0.2277.112

### Steps to reproduce the behavior

`<p-tag value="Tag 1"></p-tag>`

### Expected behavior

`p-tag` should receive keyboard focus via tabbing

**Comment by SoyDiego** [comment]
Hi @akashuwtw you should add `tabindex="0"`. In the docs you can find this information:

```
In case badges need to be tabbable, tabIndex can be added to implement custom key handlers. 
```

Here you have a example working:

![tab](https://github.com/primefaces/primeng/assets/19764334/4fbea56e-661b-49b6-8d1c-99639e5f1636)

**Comment by akashuwtw** [comment]
Thanks @SoyDiego. It worked.

**Comment by SoyDiego** [comment]
> Thanks @SoyDiego. It worked.

Perfect. You can close the issue if it's solved :)
